packages/react-native-playground/components/Activity/ActivityItem/index.stories.tsx

```diff
@@ -0,0 +1,182 @@
+import React from "react";
+import { ScrollView, View, Text } from "react-native";
+import type { Meta, StoryObj } from "@storybook/react";
+import {
+  ActivityItem,
+  ActivityItemSkeleton,
+} from "@factorialco/factorial-one-react-native";
+import { AppIcons } from "@factorialco/factorial-one-react-native";
+
+// Helper function to format dates
+const formatDate = (date: Date): string => {
+  const now = new Date();
+  const diffMs = now.getTime() - date.getTime();
+  const diffMins = Math.floor(diffMs / (1000 * 60));
+
+  if (diffMins < 1) return "Just now";
+  if (diffMins < 60) return `${diffMins} min ago`;
+
+  const diffHours = Math.floor(diffMins / 60);
+  if (diffHours < 24) return `${diffHours} hours ago`;
+
+  const options: Intl.DateTimeFormatOptions = {
+    month: "short",
+    day: "numeric",
+  };
+  return date.toLocaleDateString(undefined, options);
+};
+
+const meta = {
+  title: "Components/ActivityItem",
+  component: ActivityItem,
+  argTypes: {
+    isUnread: {
+      control: "boolean",
+    },
+    onPress: { action: "pressed" },
+  },
+  args: {
+    id: "1",
+    date: formatDate(new Date()),
+    title: "Your time off request has been approved",
+    description: "René Galindo has approved your time off request",
+    category: "Time off",
+    isUnread: false,
+    onPress: (id: string) => console.log(`ActivityItem ${id} pressed`),
+  },
+  decorators: [
+    (Story) => (
+      <View className="flex-1 p-4">
+        <Story />
+      </View>
+    ),
+  ],
+} satisfies Meta<typeof ActivityItem>;
+
+export default meta;
+
+type Story = StoryObj<typeof meta>;
+
+export const Default: Story = {
+  args: {
+    id: "1",
+    date: formatDate(new Date()),
+    title: "Your time off request has been approved",
+    description: "René Galindo has approved your time off request",
+    category: "Time off",
+    icon: AppIcons.Calendar,
+    isUnread: false,
+  },
+};
+
+export const Unread: Story = {
+  args: {
+    ...Default.args,
+    id: "2",
+    isUnread: true,
+  },
+};
+
+export const WithoutIcon: Story = {
+  args: {
+    ...Default.args,
+    id: "3",
+    icon: undefined,
+  },
+};
+
+export const WithoutDescription: Story = {
+  args: {
+    ...Default.args,
+    id: "4",
+    description: undefined,
+  },
+};
+
+export const ActivityItemList: Story = {
+  args: {
+    // Adding default args to fix type error
+    id: "1",
+    date: formatDate(new Date()),
```

Comment:
Please don't use variable values as they will cause false positive in visual tests